src/python/test/test_dxclient.py

```diff
@@ -128,12 +128,12 @@ def test_without_project_context(self):
             self.assertNotIn('DX_PROJECT_CONTEXT_ID', run('dx env --bash'))
         self.assertIn('DX_PROJECT_CONTEXT_ID', run('dx env --bash'))
 
-    @unittest.skipUnless(testutil.TEST_ENV, 'skipping test that would clobber your local environment')
-    def test_without_auth(self):
-        self.assertIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
-        with without_auth():
-            self.assertNotIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
-        self.assertIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
+    # @unittest.skipUnless(testutil.TEST_ENV, 'skipping test that would clobber your local environment')
+    # def test_without_auth(self):
+    #     self.assertIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
+    #     with without_auth():
+    #         self.assertNotIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
+    #     self.assertIn('DX_SECURITY_CONTEXT', run('dx env --bash'))
```

Comment:
why commented not deleted?

Reply:
I asked - is this test important? nobody knows..

Reply:
you think I should better delet?

Reply:
I think it's better to add a comment about it

Reply:
add a comment to a comment? NOISCE